### DISCUSSION TITLE
Setup model without interacting with Dialog box

### DISCUSSION THREAD

**Discussion by usyed15181** [pycaret/pycaret#1522] [General]
Hi, Can you avoid the inferred data type dialogue box after model setup. 

I want the put whole process from 'setup' to 'assign_model' be put inside a function.

**Comment by moezali1** [comment]
Yes you can achieve that by passing silent = True within the setup function.

**Reply by usyed15181** [reply]
Thank you sir! It worked. I missed it some how while going through documentation :/
